Tools.Test.Database/Model/Tasks/CreateOrganizationTask.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using Core.DomainModel;
+using Core.DomainModel.Organization;
+using Tools.Test.Database.Model.Extensions;
+
+namespace Tools.Test.Database.Model.Tasks
+{
+    public class CreateOrganizationTask : DatabaseTask
+    {
+        private readonly string _orgName;
+        private readonly int _organizationType;
+
+        public CreateOrganizationTask(string connectionString, string organizationType, string orgName) : base(connectionString)
```

Comment:
Hellere parse det helt ude hvor du parser dine parametre i stedet for herinde - ellers bliver tingene blandet sammen